src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsole.cs

```diff
@@ -654,9 +647,12 @@ public void SetExecutionMode(bool isExecuting)
 
             if (!isExecuting)
             {
-                HideProgress();
-
-                VsUIShell.UpdateCommandUI(0 /* false = update UI asynchronously */);
+                ThreadHelper.JoinableTaskFactory.Run(async delegate
+                {
+                    await WriteProgressAsync(string.Empty, 100);
+                    var vsUiShell = await AsyncServiceProvider.GlobalProvider.GetServiceAsync<IVsUIShell>();
+                    vsUiShell.UpdateCommandUI(fImmediateUpdate: 0 /* false = update UI asynchronously */);
+                });
```

Comment:
like my previous comment. It's well intentioned, but it's not actually helping.

Reply:
Ok. Reverted this change.